server.js

```diff
@@ -27,6 +30,72 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+// {
+//   "bookID": 78,
+//   "title": "Annals of the Former World",
+//   "authors": "John McPhee",
+//   "average_rating": 4.33,
+//   "isbn": 374518734,
+//   "isbn13": 9780374518738,
+//   "language_code": "eng",
+//   "num_pages": 720,
+//   "ratings_count": 2966,
+//   "text_reviews_count": 211
+// },
+
+app.get('/allBooks', (req, res) => {
+  res.json(booksData)
+})
+
+// path variable/path parameter
+
+// app.get('authors/:author', (req, res) => {
+//   const authors = req.params.authors
+//   const nameOfAuthor = booksData.filter((item) => item.authors === authors)
+//   res.json(nameOfAuthor)
+// })
+
+
+// path variable/path parameter
+// app.get('/book/:id', (req, res) => {
+//   const bookId = req.params.id
+//   const book = booksData.filter(item => item.bookID === +bookId)
+//   res.json(book) §§
+// })
+
+// query parameter
+app.get('/author', (req, res) => {
```

Comment:
Since this is filtering from the booksData, make these queries in the books endpoint instead of creating a new one.